dashboard/app/models/script.rb

```diff
@@ -1087,10 +1087,10 @@ def alternate_script(user)
   # @return {AssignableInfo} with strings translated
   def assignable_info
     info = ScriptConstants.assignable_info(self)
-    info[:name] = I18n.t("#{info[:name]}_name", default: info[:name])
+    info[:name] = I18n.t("data.script.name.#{info[:name]}.title", default: info[:name])
```

Comment:
While it looks like the script tests added in the previous PR may still pass, they will only be testing the "default" case here. It could be worth adding a test that the i18n strings are getting picked up. There is an example of where we do this in pegasus tests: https://github.com/code-dot-org/code-dot-org/blob/310f220465d5290a970e22089d9cf3cde5c224aa/pegasus/test/test_section_api_helpers.rb#L356-L362

I'd recommend at least giving this a quick try.

Reply:
great point! i worried about adding a test that relied on a specific script, but i suppose CS Fundamentals won't be going away any time soon 😆